client/extensions/woocommerce/components/order-status/index.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { localize } from 'i18n-calypso';
+import React from 'react';
+
+function OrderStatus( { showPayment = true, showShipping = true, status, translate } ) {
+	const classes = `order-status__item is-${ status }`;
+	let paymentLabel;
+	let shippingLabel;
+	switch ( status ) {
+		case 'pending':
+			shippingLabel = translate( 'New order' );
+			paymentLabel = translate( 'Payment pending' );
+			break;
+		case 'processing':
+			shippingLabel = translate( 'New order' );
+			paymentLabel = translate( 'Paid in full' );
+			break;
+		case 'on-hold':
+			shippingLabel = translate( 'On hold' );
+			paymentLabel = translate( 'Payment pending' );
+			break;
+		case 'completed':
+			shippingLabel = translate( 'Fulfilled' );
+			paymentLabel = translate( 'Paid in full' );
+			break;
+		case 'cancelled':
+			paymentLabel = translate( 'Cancelled' );
+			break;
+		case 'refunded':
+			paymentLabel = translate( 'Refunded' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **37** times:
`translate( 'Refunded To' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/store-orders-new-status-component)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).